dali/base/dadfs.cpp

```diff
@@ -3308,7 +3309,7 @@ class CDistributedFile: public CDistributedFileBase<IDistributedFile>
                 throw exceptions.getClear();
         }
     }
-    bool removePhysicalPartFiles(IFileDescriptor *fileDesc, IMultiException *mexcept)
+    bool removePhysicalPartFiles(IFileDescriptor *fileDesc, IMultiException *mexcept, unsigned parallelPhysicalDeletes=NotFound)
```

Comment:
I think something like 'numberOfParallelPhysicalDeletes' or 'numberOfParallelPhysicalDeleteThreads' would be better than 'parallelPhysicalDeletes'
I think the 'NotFound' is not the best self documented default value for 'parallelPhysicalDeletes'. Something like 'UnSet' or 'NotSet' would be better (or simple 0 if the parameter renamed to 'numberOfParallel...')